main.ts

```diff
@@ -36,6 +38,28 @@ export default class NLSyntaxHighlightPlugin extends Plugin {
 		await this.saveData(this.settings);
 	}
 
+	async addCommands() {
```

Comment:
I don't think `addCommands()` itself needs to be async (since `this.addCommand` is synchronous) - it's just the callback function that does